index.js

```diff
@@ -12,14 +14,23 @@ var fs = require('fs');
  * @returns {Stream} the formatted code
  */
 function clangFormat(file, enc, style, done) {
-  return spawnClangFormat(['-style=' + style, file.path], done, ['ignore', 'pipe', process.stderr])
-      .stdout;
+  var args = ['-style=' + style, file.path];
+  return spawnClangFormat(args, done, ['ignore', 'pipe', process.stderr]).stdout;
 }
 
 /**
- * Spawn the clang-format binary with given arguments
+ * Spawn the clang-format binary with given arguments.
  */
 function spawnClangFormat(args, done, stdio) {
+  // WARNING: This function's interface should stay stable across versions for the cross-version
+  // loading below to work.
+  if (args.indexOf('-version') !== -1 || args.indexOf('--version') !== -1) {
+    // Print our version.
+    // This makes it impossible to format files called '-version' or '--version'. That's a feature.
+    // minimist & Co don't support single dash args, which we need to match binary clang-format.
```

Comment:
should minimist still be added in the package.json? I don't see where the parseArgs is used here.